packages/rich-text/src/component/use-anchor-ref.js

```diff
@@ -10,7 +10,7 @@ import deprecated from '@wordpress/deprecated';
 import { getActiveFormat } from '../get-active-format';
 
 /** @typedef {import('@wordpress/element').RefObject} RefObject */
```

Comment:
Yup, this is because when importing a generic in JSDoc we need to specifically declare it as one, listing all of its parameters. See https://github.com/microsoft/TypeScript-Handbook/issues/1106

I've added a [commit](https://github.com/WordPress/gutenberg/pull/49651/commits/15b401a4c0e4335e6bb7d6540c2c1b13aa3d77a1) to fix that. Let me know what you think.